src/commands/login.yml

```diff
@@ -15,10 +15,39 @@ parameters:
     default: VAULT_SECRET_ID
 
 steps:
+  - run:
+      name: Checking for Vault environment variables
+      command: |
+        echo "Checking for the existence of Vault Environment Variables"
+        [ -z "$<<parameters.vault-addr>>" ] && ERROR=true && \
+          echo "ERROR: <<parameters.vault-addr>> env is not set." || \
+          echo "Found <<parameters.vault-addr>>: $<<parameters.vault-addr>>"
+        [ -z "$<<parameters.vault-role-id>>" ] && ERROR=true && \
+          echo "ERROR: <<parameters.vault-role-id>> env is not set." || \
+          echo "Found <<parameters.vault-role-id>>: $<<parameters.vault-role-id>>"
+        [ -z "$<<parameters.vault-secret-id>>" ] && ERROR=true && \
+          echo "ERROR: <<parameters.vault-secret-id>> env is not set." || \
+          echo "Found <<parameters.vault-secret-id>>: **HIDDEN**"
+
+        echo
+        if [[ -n "$ERROR" ]]; then
+          echo "ERROR:"
+          echo
+          echo "You should check your projects or context Environment Variables page to see if you set <<parameters.vault-addr>>,"
+          echo "<<parameters.vault-role-id>> and <<parameters.vault-secret-id>>"
+          echo
+          exit 1
+        else
+          echo "Environment variables look all good."
```

Comment:
Nit: here we have "look all good." and above "look all clean."
I think we should have consistent messages.
E.g. "environment variables passed initial check"

Reply:
How about what I changed it to, they are two different checks, one checks if the variables are not there and the other makes sure they are.

Reply:
👍 looks great!